### PR TITLE
Add a notification when ready to merge

### LINKED ISSUE #52: Notifications on ready-to-merge
It would be nice if we sent out a notification to openssl-commits when a PR becomes ready-to-merge, mentioning the reviewer names in the notification as a reminder that they might want to come back and merge the PR

### PR DESCRIPTION
**Pull request by iamamoose**
closes #52

### CODE REVIEW

**Comment by iamamoose** [comment]
as tested on https://github.com/openssl/openssl/pull/11027

File: github-approve-label-workflow/github-approve-label-workflow.py [hunk 1]
```diff
@@ -60,6 +60,13 @@ def movelabeldonetoready(issue):
     res = requests.post(url, data=json.dumps(newlabel), headers=headers)
     if (res.status_code != 200):
         print("Error adding label", res.status_code, res.content)
+        return
+    newcomment = {"body":"This issue is ready to merge"}
```
Comment (paulidale) [inline github-approve-label-workflow/github-approve-label-workflow.py]: s/issue/pull request/ ????

Or just say "This is ready to merge" ?

Comment (iamamoose) [reply github-approve-label-workflow/github-approve-label-workflow.py]: everything is an issue, but ok :)

Comment (paulidale) [reply github-approve-label-workflow/github-approve-label-workflow.py]: The URL I clicked through on has "pull" not "issues" :)

**Pull request merged**